drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -984,7 +1024,13 @@ function DragAndDropBlock(runtime, element, configuration) {
             data: '{}'
         }).done(function(data){
             state.attempts = data.attempts;
-            state.overall_feedback = data.feedback;
+            state.feedback = data.feedback;
+            state.overall_feedback = data.overall_feedback;
+
+            if (data.feedback) {
+                state.last_action_correct = false;
```

Comment:
@e-kolpakov I'm not sure if `last_action_correct` is the most appropriate name in this context. The last action that the user performed was to submit an answer; we might decide that that doesn't count, but the last "drop item" action before that might still have been correct. Rename to `all_items_correct`, perhaps?